05_test.sh

```diff
@@ -127,7 +127,7 @@ check_bmh_status(){
 #
 # Usage ip_in_range ip start end
 ip_in_range() {
-  python ip_range_check.py "$1" "$2"
+  python3 ip_range_check.py "$1" "$2"
```

Comment:
This should be transparent due to https://github.com/metal3-io/metal3-dev-env/blob/master/centos_install_requirements.sh#L35 - do we need something similar for ubuntu perhaps?

Reply:
The problem is not for centos8. It was running with python3, and that's what we want. But for others, such as Ubuntu, and Centos7, we need to be specific as long as there is python2